processor/spanmetricsprocessor/config.go

```diff
@@ -45,4 +51,15 @@ type Config struct {
 	// The dimensions will be fetched from the span's attributes. Examples of some conventionally used attributes:
 	// https://github.com/open-telemetry/opentelemetry-collector/blob/main/model/semconv/opentelemetry.go.
 	Dimensions []Dimension `mapstructure:"dimensions"`
+
+	AggregationTemporality string `mapstructure:"aggregation_temporality"`
+}
+
+// GetAggregationTemporality converts the string value given in the config into a MetricAggregationTemporality.
+// Returns cumulative, unless delta is correctly specified.
+func (c Config) GetAggregationTemporality() pdata.MetricAggregationTemporality {
```

Comment:
Is it okay to assume that anything that doesn't match `delta` is to be considered `cumulative` ?

Reply:
I think so, since the default is cumulative.